Garza/taller ros

davidogarzas opened the pull request:
Ya arregle lo de los overflows de las slides

G00Z-G00Z committed:
Espacio para taller

G00Z-G00Z committed:
Edy ROS notes

G00Z-G00Z committed:
Added to introducción a Linux

G00Z-G00Z committed:
Added link to home

G00Z-G00Z committed:
Added resources

G00Z-G00Z committed:
Added python documentation file

G00Z-G00Z committed:
Finished Introduccion a Linux

G00Z-G00Z committed:
Added some slides

G00Z-G00Z committed:
Minicurso de Linux Presentacion

G00Z-G00Z committed:
Virtual machine setup

G00Z-G00Z committed:
Added master class link

G00Z-G00Z committed:
Added ROS as a taller

G00Z-G00Z committed:
Added vm setup

G00Z-G00Z committed:
Added drawing of ejercicio

G00Z-G00Z committed:
Ignored all svg

G00Z-G00Z committed:
Added reference to overview

G00Z-G00Z committed:
Merge branch 'main' into edy/taller-ros

david-villanueva-02 committed:
Explicación Topics y messages

david-villanueva-02 committed:
Comenzar creación de nodo

david-villanueva-02 committed:
Avances

david-villanueva-02 committed:
Avances1.0

david-villanueva-02 committed:
Avances2.0

Checar tamaño del bloque de código, se puede cambiar?

davidogarzas committed:
Agregar workspaces

Slides workspaces

davidogarzas committed:
Turtlesim

Agregue turtlesim y turtlechaser

davidogarzas committed:
Agregar parte Edy

david-villanueva-02 committed:
Cambios

davidogarzas committed:
Fixed slides overflow

david-villanueva-02 committed:
Merge branch 'setup-ejercicio' into villa/taller-ros

david-villanueva-02 committed:
Añadir boceto 1 ejemplo práctico

davidogarzas committed:
merge branch biya packages nodes

davidogarzas committed:
merge branch de biya (agregue packages-nodes)

davidogarzas committed:
Agregar código de node template

davidogarzas committed:
Arregle overflow de slides de biya

G00Z-G00Z commented on talleres/Taller ROS/Presentacion Taller ROS.md:
```diff
@@ -0,0 +1,317 @@
+---
+
+<!-- .slide: data-auto-animate -->
+### ⚡🚢Navegación En File System
```
Checa aquí como se pueden poner notas

G00Z-G00Z commented on talleres/Taller ROS/Presentacion Taller ROS.md:
```diff
@@ -0,0 +1,317 @@
+- `touch <filename>`: Crea un archivo en el `pwd`
+- `rm <filename>`: Elimina un archivo
+
+note:
```
Aqui empieza la nota y abajo no se renderiza

G00Z-G00Z commented on talleres/Taller ROS/Presentacion Taller ROS.md:
```diff
@@ -0,0 +1,317 @@
+Ve hasta abajo del documento y se deben ver las siguientes 3 líneas:
+
+---
+![[Pasted image 20231101215019.png]]
```
No peques screenshot del código, porque no lo podemos copiar cuando estemos presentando

Tienes de 2: O puedes escribir el código entre corchetas

```bash
if [ -z ... ] 
echo "hello world" 
fi
```

O puedes dejar la screenshot y poner *notas* en la slide

Para poner notas, puedes ponter `note: ` en una linea, y luego puedes escribir abajo

Por ejemplo checar la slide de "⚡🚢Navegación En File System" (te deje un comentario arriba)

G00Z-G00Z reviewed (changes requested):
Muchísima mejora!! El layout y la información ya quedarón. Nomás un detalle con una ss de código, que es básicamente que necesitamos el texto para poder copy paste durante la presentación (por si lo ocupamos pasar en caliente)

Además, para el *turtlesim*, maybe deberías poner más imagenes. Como poner el comando que corre y el ss del turtle sim después

davidogarzas committed:
Agregue seccion para publishers

Agregue seccion para publishers, hay algunos bloques de codigo que me gustaria que fueran mas grandes
Tambien hice unas pequeñas correcciones en la seccion de turtlesim (agregue imgs y quite un ss de codigo)

davidogarzas committed:
Agregar paso a paso creacion de un nodo publisher

davidogarzas committed:
Agregue subscriber
